docs/user/alerting/alerting-getting-started.asciidoc

```diff
@@ -150,65 +134,4 @@ Functionally, {kib} alerting differs in that:
 At a higher level, {kib} alerting allows rich integrations across use cases like <<xpack-apm,*APM*>>, <<metrics-app,*Metrics*>>, <<xpack-siem,*Security*>>, and <<uptime-app,*Uptime*>>.
 Pre-packaged *rule types* simplify setup and hide the details of complex, domain-specific detections, while providing a consistent interface across {kib}.
 
-[float]
```

Comment:
Moved to its own page

Reply:
@gchaps Addressed your comments in this commit: https://github.com/elastic/kibana/pull/101323/commits/74c50dfd16fa7bcb8ab76b0b59b7e9427b0b6e40

I changed the paragraph to be a list...is that clearer? The condition and schedule can have different times.

The Alerting prerequisites section is there temporarily for cross doc link compatibility. Once this PR is merged, I can make a PR to update the observability doc that links to the prerequisites section, and then I will do a cleanup PR here to remove it.